File: src/test/java/edu/colorado/plv/fixr/SlicingTestCF.java
package edu.colorado.plv.fixr;

import org.junit.Test;

public class SlicingTestCF extends TestSlicing {
	static final String CLASS_NAME = "slice.TestControlFlow";
	static final String RES_CLASS_NAME = "slice.TestControlFlowRes";		
	
	public String getTestClassName()
	{
		return CLASS_NAME;
	}
	public String getResClassName()
	{
		return RES_CLASS_NAME;
	}
		
	@Test
	public void t1(){testSlice("testSequence01");}

	@Test
	public void t2(){testSlice("testSequence02");}
	
	@Test
	public void t3(){testSlice("testSequence03");}
	
	@Test
	public void t4(){testSlice("testConditional01");}

	@Test
	public void t5(){testSlice("testConditional02");}
	
	@Test
	public void t6(){testSlice("testConditional03");}
	
	@Test
	public void t7(){testSlice("testLoop01");}
	@Test
	public void t8(){testSlice("testLoop02");}
	@Test
	public void t9(){testSlice("testLoop03");}
}

File: src/test/scala/edu/colorado/plv/fixr/tests/slicing/SlicingTestCF.scala
package edu.colorado.plv.fixr.tests.slicing

class SlicingTestCF extends TestSlicing("./src/test/resources/jimple",
    "slice.TestControlFlow", "slice.TestControlFlowRes") {
  
  override def getPackages() : List[String] = {
    return List[String]("java.lang.Math")
  }

  test("testSequence01") {testSlice("testSequence01")}
  test("testSequence02") {testSlice("testSequence02")}
  test("testSequence03") {testSlice("testSequence03")}
  test("testConditional01") {testSlice("testConditional01")}
  test("testConditional02") {testSlice("testConditional02")}
  test("testConditional03") {testSlice("testConditional03")}
  test("testLoop01") {testSlice("testLoop01")}
  test("testLoop02") {testSlice("testLoop02")}
  test("testLoop03") {testSlice("testLoop03")}
}